### ISSUE TITLE
Add rule S4798 (bool-param-default	4798

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarJS#1531]
**I want to request a feature.**

**I want to report a bug.**

**SonarJS version:** <br>
**SonarQube version:** <br>
**SonarLint version:** <br>

**Rule key:** 

**Reproducer** 
```
// minimal reproducer if relevant
console.log("Hello, world"); // False-positive here
```

**Logs**
```
// logs of analysis if relevant
```

**Expected behavior**

**I want to provide the following feedback.**

**Rule key:** 

**Example**
